stdlib/public/Concurrency/AsyncSequence.swift

```diff
@@ -85,6 +248,20 @@ extension AsyncSequence {
 
 @available(SwiftStdlib 5.5, *)
 extension AsyncSequence where Element: Equatable {
+  /// Returns a Boolean value that indicates whether the asynchronous sequence
+  /// contains the given element.
+  ///
+  /// In this example, an asynchronous sequence called `Counter` produces `Int`
+  /// values from `1` to `10`. The `contains(_:)` method checks to see whether
+  /// the sequence produces the value `5`:
+  ///
+  ///     let containsFive = await Counter(howHigh: 10)
+  ///         .contains(5)
+  ///     print(containsFive)
```

Comment:
Unlike the other examples, you don't show the printed output here.

Reply:
Added output.